Neos.Neos/Configuration/Settings.yaml

```diff
@@ -278,7 +278,7 @@ Neos:
           title: Live
         inPlace:
           isEditingMode: true
-          isPreviewMode: false
+          isPreviewMode: true
```

Comment:
Your thesis:

defaultEditPreviewMode should have both to true

Reply:
One could rephrase this as the ui has to switch to edit / preview depending on what the defaultMode is.